### PR TITLE
CB-5678 DP cli should have the possibility to chose DL platform version

### PR DESCRIPTION
**Pull request by sodre90**

### CODE REVIEW

File: dataplane/cmd/sdx.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ func init() {
...
+				Flags:       fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentName, fl.FlClusterShape, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional, fl.FlCloudStorageManagedIdentityOptional, fl.FlRuntimeOptional, fl.FlCidrOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlWithExternalDatabaseOptional).AddAuthenticationFlags().Build(),
 				Action:      sdx.CreateSdx,
 				BashComplete: func(c *cli.Context) {
 					for _, f := range fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentName, fl.FlEnvironmentName, fl.FlClusterShape, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional, fl.FlCloudStorageManagedIdentityOptional, fl.FlCidrOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlWithExternalDatabaseOptional).AddAuthenticationFlags().Build() {
```
Comment (bbihari) [inline dataplane/cmd/sdx.go]: You should add the new flag to the 'bashcomplete' too.

File: dataplane/cmd/sdxinternal.go [hunk 2]
```diff
@@ -18,7 +18,7 @@ func init() {
 				Usage:       "creates a new SDX cluster",
 				Description: `basic SDX cluster creation`,
 				Before:      cf.CheckConfigAndCommandFlagsWithoutWorkspace,
-				Flags:       fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentName, fl.FlInputJson, fl.FlWithExternalDatabaseOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional).AddAuthenticationFlags().Build(),
+				Flags:       fl.NewFlagBuilder().AddResourceDefaultFlags().AddFlags(fl.FlEnvironmentName, fl.FlInputJson, fl.FlWithExternalDatabaseOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlRuntimeOptional, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional).AddAuthenticationFlags().Build(),
 				Action:      sdx.CreateSdx,
 				BashComplete: func(c *cli.Context) {
 					for _, f := range fl.NewFlagBuilder().AddResourceFlagsWithOptionalName().AddFlags(fl.FlEnvironmentName, fl.FlEnvironmentName, fl.FlInputJson, fl.FlWithExternalDatabaseOptional, fl.FlWithoutExternalDatabaseOptional, fl.FlCloudStorageBaseLocationOptional, fl.FlCloudStorageInstanceProfileOptional).AddAuthenticationFlags().Build() {
```
Comment (bbihari) [inline dataplane/cmd/sdxinternal.go]: You should add the new flag to the 'bashcomplete' too.

File: dataplane/cmd/sdx.go (hunk 1 above)
Comment (sodre90) [reply dataplane/cmd/sdx.go]: Thanks, done!

File: dataplane/cmd/sdxinternal.go (hunk 2 above)
Comment (sodre90) [reply dataplane/cmd/sdxinternal.go]: Thanks, done!

**Pull request merged**